test/system/departments_test.rb

```diff
@@ -118,6 +123,56 @@ def setup
       end
     end
     click_button 'Update Department'
+    assert page.has_text? 'Department was successfully updated.'
     refute page.has_text? delete_phone
   end
+
+  test 'As Department Manager there is a link in the Navbar to his department' do
+    login_as @department_manager
+    visit profile_path(@department_manager.profile.id)
+    assert page.has_link? 'Department',
+      href: department_path(@department_manager.department.first.id)
+  end
+
+  test "Department Managers can update their department's fields" do
+    login_as @department_manager
+    visit edit_department_path(@department_manager.department.first.id)
+    refute page.has_select? 'User'
+    fill_in 'Name', with: 'Name changed'
+    fill_in 'Street', with: 'Street changed'
+    fill_in 'Extended address', with: 'Extended address changed'
+    fill_in 'Zip', with: 'Zip changed'
+    fill_in 'City', with: 'City changed'
+    within '#phones' do
```

Comment:
no, but this
```ruby
within '#phones' do
```
shouldn't be there at all, as the `#phones` field is filled in at line 155 ;)